static/js/controllers/configuration-upgrade.js

```diff
@@ -0,0 +1,110 @@
+var PouchDB = require('pouchdb-browser');
```

Comment:
When you say it doesn't work with angular, you mean angular can't tell when it needs to run pouchdb queries again right? Does that matter here? Specifically, what features are we missing by using it directly?

I ask because I think the idea would be at some point we'd split this out of our codebase entirely (along with the stuff in medic-api). It's just easier to have it inline for now.